test/configuration_mixin_test.rb

```diff
@@ -42,4 +42,24 @@
       assert_equal "kasket-#{protocol}/R#{ar_version}/posts/version=#{POST_VERSION}/", Post.kasket_key_prefix
     end
   end
+
+  describe "kasket_ttl" do
+    describe "with an explicit TTL" do
+      it "returns the local TTL" do
+        assert_equal 5.minutes, ExpiringComment.kasket_ttl
+      end
+    end
+
+    describe "without an explicit TTL" do
+      it "falls back to the global" do
+        begin
+          previous = Kasket::CONFIGURATION[:default_expires_in]
+          Kasket::CONFIGURATION[:default_expires_in] = 86401
+          assert_equal 5.minutes, DefaultComment.kasket_ttl
```

Comment:
kinda suspicious example that this also has 5 minutes ... maybe do `default_expires_in = 123` and then assert 123 so it's obviously a bogus number

Reply:
Yeah, that's a copy paste mistake.